iroha-cli/interactive/interactive_query_cli.hpp

```diff
@@ -150,11 +158,20 @@ namespace iroha_cli {
       // Query generator for new queries
       iroha::model::generators::QueryGenerator generator_;
 
-      // Logger
-      logger::Logger log_;
-
       // Crypto provider
       std::shared_ptr<iroha::model::ModelCryptoProvider> provider_;
+
+      /// Logger manager for GrpcResponseHandler
+      logger::LoggerManagerTreePtr responce_handler_log_manager_;
+
+      /// Logger for PbQueryFactory
+      logger::LoggerPtr pb_qry_factory_log_;
+
+      /// Logger for JsonQueryFactory
```

Comment:
Are you sure that these comments are useful? It looks like they can be derived from type and variable name.